number.py

```diff
@@ -26,6 +27,13 @@
     "light_auto_motion_timeout": NumberMode.SLIDER,
 }
 
+UNITS = {
+    "return_to_auto_timeout": TIME_SECONDS,
+    "motion_sense_timeout": TIME_SECONDS,
+    "light_return_to_auto_timeout": TIME_SECONDS,
+    "light_auto_motion_timeout": TIME_SECONDS,
+}
+
 FAN_NUMBER_DESCRIPTIONS = (
     NumberEntityDescription(
         key="return_to_auto_timeout",
```

Comment:
```suggestion
        key="return_to_auto_timeout",
        unit_of_measurement=TIME_SECONDS,        
```

I'm pretty sure this will work instead

Reply:
Ah yeah. I assume this is the preferred way. I'll redo the PR.